### PR TITLE
Js/week5/Min Ko

### PR DESCRIPTION
**Pull request by minwwgo**

### CODE REVIEW

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 1]
```diff
@@ -57,8 +78,42 @@ function exerciseTwo(shopping) {
...
+      pictureBook.setAttribute('src', 'https://image.ebooks.com/previews/209/209748/209748258/209748258-hq-168-80.jpg')
+    }
+    
+    pictureBook.classList.add('photo')
+    if ( book.alreadyRead){
+      bookList.style.background = 'green'
+    }else{
+      bookList.style.background = 'red'
+    }
+
```
Comment (ChrisOwen101) [inline week-5/Homework/mandatory/2-exercises/exercises.js:112]: Could you remove the blank space here?

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 2]
```diff
@@ -57,8 +78,42 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
+
+
 function exerciseThree(books) {
   //Write your code in here
+  const holder = document.querySelector("#content");
+  const bookListHolder = document.createElement('ul');
+    holder.append(bookListHolder)
```
Comment (ChrisOwen101) [inline week-5/Homework/mandatory/2-exercises/exercises.js:87]: Some formatting errors here - are you using this?

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 3]
```diff
@@ -57,8 +78,42 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
+
```
Comment (ChrisOwen101) [inline week-5/Homework/mandatory/2-exercises/exercises.js:81]: I'm getting this error when I try to run this file

```
Uncaught SyntaxError: Unexpected end of input
```
So your code won't compile. Can you please fix this?

Comment (minwwgo) [reply week-5/Homework/mandatory/2-exercises/exercises.js:76]: i fixed it and please run again

File: week-5/Homework/mandatory/2-exercises/exercises.js (hunk 2 above)
Comment (minwwgo) [reply week-5/Homework/mandatory/2-exercises/exercises.js]: i changed all code format in prettier.

**Pull request closed without merging**